pageserver/src/tenant/timeline.rs

```diff
@@ -1541,7 +1541,9 @@ impl Timeline {
                     lsn,
                 )?;
 
+                info!("blabla for");
```

Comment:
NIT: Simpler to mark such PRs as drafts if you have debugging enabled :slightly_smiling_face: